classes/components/forms/highlight/HighlightForm.php

```diff
@@ -42,10 +44,10 @@ public function __construct(string $action, string $baseUrl, string $temporaryFi
         $this->locales = $this->getLocales($context);
 
         $this->addField(new FieldRichText('title', [
-                'label' => __('common.title'),
-                'isMultilingual' => true,
-            ]))
-            ->addField(new FieldRichText('description', [
+            'label' => __('common.title'),
+            'isMultilingual' => true,
+        ]))
+            ->addField(new FieldRichTextarea('description', [
                 'label' => __('common.description'),
                 'isMultilingual' => true,
```

Comment:
Probably doesn't matter but we can also add the 'size' => 'large', here as we do for the other versions.
```suggestion
                'isMultilingual' => true,
                'size' => 'large',
```

That would only be required if we enable the feature flag on v3.4, but I'm not sure.